py_trees/composites.py

```diff
@@ -731,14 +731,12 @@ def tick(self) -> typing.Iterator[behaviour.Behaviour]:
                     new_status = common.Status.SUCCESS
                     self.current_child = self.children[-1]
             elif type(self.policy) is common.ParallelPolicy.SuccessOnOne:
-                successful = [
-                    child
-                    for child in self.children
-                    if child.status == common.Status.SUCCESS
-                ]
-                if successful:
-                    new_status = common.Status.SUCCESS
-                    self.current_child = successful[-1]
+                for index in range(len(self.children) - 1, -1, -1):
+                    child = self.children[index]
```

Comment:
You can make this a little cleaner by doing:

```python
for child in reversed(self.children):
   # do the thing
```

Reply:
You're right. Thank you for your advice

Reply:
But I haven't done much pr. Do I need to submit it again?

Reply:
Not at all! You can just add a new commit to this to this branch!

Reply:
I have already submitted it, thanks again